src/test/kotlin/uk/gov/justice/digital/hmpps/whereabouts/services/VideoLinkMigrationServiceTest.kt

```diff
@@ -15,21 +24,26 @@ import uk.gov.justice.digital.hmpps.whereabouts.repository.VideoLinkAppointmentR
 import uk.gov.justice.digital.hmpps.whereabouts.repository.VideoLinkBookingEventRepository
 import uk.gov.justice.digital.hmpps.whereabouts.repository.VideoLinkBookingRepository
 import uk.gov.justice.digital.hmpps.whereabouts.services.court.VideoLinkMigrationService
+import uk.gov.justice.digital.hmpps.whereabouts.services.court.events.OutboundEvent
+import uk.gov.justice.digital.hmpps.whereabouts.services.court.events.OutboundEventsService
 import uk.gov.justice.digital.hmpps.whereabouts.utils.DataHelpers.Companion.makeVideoLinkBooking
 import java.lang.IllegalArgumentException
 import java.time.LocalDate
 import java.time.LocalDateTime
 import java.util.Optional
 
+@ContextConfiguration(classes = [TestAsyncConfiguration::class])
```

Comment:
May be I am missing something but are you sure this is needed for the unit test on the service?

There will be no async at this level, the aysnc config will not be autowired in this kind of test?

Reply:
I think its a belt - and - braces  so will remove and I would imagine it would work the same - due to the test-configured bean.

Reply:
If it is was an integration/resource test then yes the test config makes sense, but this isn't one of those.

The aysnc stuff won't be active here in a unit test.

Reply:
Yes, you're right. Will remove.

Reply:
No context loaded here :-)

Reply:
Done